### ISSUE TITLE
Investigate a key splitting algorithm or service for new Brave Wallet account creation

### ISSUE THREAD

**Issue by bbondy** [brave/brave-browser#19538]
Not to replace the current way, but it would make sense to never tell a user the seed phrase and instead split their seed phrase across 1 or more services + the local device. In this way it would eliminate a whole class of issues where users lose funds. We could also do things like add 2fa for withdraws at this point.

Torus login does this and they might be an option to use instead of rolling our own.

**Comment by Tynael** [comment]
Great idea!

Also 2FA is more than welcomed at this point.

**Comment by fmarier** [comment]
GitHub used to offer the ability to [store some recovery tokens on Facebook](https://github.blog/2017-01-30-recover-accounts-elsewhere/), but that is [now deprecated](https://docs.github.com/en/authentication/securing-your-account-with-two-factor-authentication-2fa/recovering-your-account-if-you-lose-your-2fa-credentials#authenticating-with-an-account-recovery-token).

**Comment by fmarier** [comment]
Obviously, we shouldn't tell users to store their recovery codes there, but this can of approach could be used if the codes are to be split and stored across multiple non-colluding providers.
